### PR TITLE
Move to a tabbed config page and tidy up material imports.

### PR DESCRIPTION
**Pull request by blaise17**

### CODE REVIEW

**Comment by psi77** [comment]
I get a vertical scrollbar which seems unnecessary.

Given that most of the time this app will be run on a fixed screen size tablet type thing, would it be better to have the save button permanently in the bottom right in the same place?  Currently it moves up and down depending on the amount of content in the current config card

File: src/app/app.module.ts [hunk 1]
```diff
@@ -5,12 +5,8 @@ import { FormsModule } from '@angular/forms';
 import { HttpClientModule } from '@angular/common/http';
 import { LayoutModule } from '@angular/cdk/layout';
 
-import { MatGridListModule } from '@angular/material/grid-list';
-import { MatCardModule } from '@angular/material/card';
-import { MatFormFieldModule } from '@angular/material/form-field';
-import { MatCheckboxModule } from '@angular/material/checkbox';
-import { MatMenuModule, MatIconModule, MatButtonModule, MatInputModule, MatRippleModule } from '@angular/material';
-import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { MatMenuModule, MatIconModule, MatButtonModule, MatInputModule, MatRippleModule, MatGridListModule, MatCardModule,
```
Comment (psi77) [inline src/app/app.module.ts]: Nice! :-)

**Comment by blaise17** [comment]
I get a vertical scrollbar on develop too, which is also unnecessary. I think maybe we'd be better with the save button in the top right to avoid movement, and then it is more obvious that it is tab agnostic.

**Comment by blaise17** [comment]
I've moved the save button to the top right so it's clearly not per tab. I'm not going to address the scroll bar issue as it already happens on deveop and is fixed in the error handling branch by the new error page wrapping stuff that also wraps the config page.

**Pull request merged**